apps/web/src/lib/monitoring/collectors/queue-depth-collector.ts

```
@@ -0,0 +1,476 @@
+    size?: number,
+    priority?: number,
+    metadata?: Record<string, unknown>
+  ): void {
+    if (!this.isRunning()) {
+      return;
+    }
+
+    const queue = this.queues.get(queueName);
+    if (!queue) {
+      this.registerQueue(queueName);
+      return this.enqueueItem(queueName, itemId, size, priority, metadata);
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)